modules/login/src/main/resources/META-INF/resources/login/js/registration.js

```diff
@@ -0,0 +1,401 @@
+var registration = function($) {
+    var orgLookupData = {};
+    var bInitialized = false;
+
+    var $formStep1, $formStep2,$formStep3,$formStep4; //Form selector cache.
```

Comment:
If these variables need to be renamed to something with 'cache' in the name, then let's do it now. Otherwise, the comment is confusing.